packages/nice-fs/README.md

```diff
@@ -0,0 +1,77 @@
+# nice-fs
+
+`nice-fs` is a minimal-dependency drop-in replacement to `fs` with changes for resiliency, promises, and convenience. It has several opinionated changes, targeted towards cross-platform CLI apps.
+
+![Usage Thumbnail](./assets/logo.png)
+
+## Async Usage (Default)
+
+`nice-fs` exposes a Promise-based API, mostly matching that of `fs.promises`, with several methods added [extra methods](#Extra-Methods).
+
+```ts
+// import {promises as fs} from 'fs'
+import fs from 'nice-fs';
+
+const fileContent = await fs.readFile('foo.txt');
+```
+
+## Sync Usage
+
+`nice-fs` exports all `fs.*Sync` Where an async version has a graceful implementation, and the synchronous version does not, the method is marked as deprecated.
+
+```ts
+// import fs from 'fs'
+import fs from 'nice-fs';
+
+const fileContent = fs.readFileSync('foo.txt');
+```
+
+## Extra Methods
+
+### `exists`
+NodeJS deprecated `fs.exists`, and removed `fs.promises.exists`. The recommendation is to instead acquire a lock to the file via `fs.open` for the duration of file-use. One-shot existence checks are still useful, and because `fs.existsSync` is not deprecated, more likely means usage of blocking synchronous APIs.
```

Comment:
>NodeJS [](http://example.com/codeflow?start=0&length=6)

It is difficult to read long lines in a text editor or in the CodeFlow. The MD file allows splitting a line into multiple lines while keeping the same paragraph.